### DISCUSSION TITLE
source.ip changes types across indices, unsearchable now

### DISCUSSION THREAD

**Discussion by Susfubar** [Security-Onion-Solutions/securityonion#5302] [Unsupported Versions] [answered]
We have been having this problem with not being able to search using source.ip. We found that source.ip is being typed as IP and Text.
![image](https://user-images.githubusercontent.com/89588946/130971194-42573c4a-0a62-4180-ace8-8fa1bd1fb2d2.png)

Searching through our elastic mappings we found that it was so-kibana-* causing the issue of type Text.

![image](https://user-images.githubusercontent.com/89588946/130971416-f0150bc0-74e0-448a-9da9-4c5b97227297.png)

We then found the log where this is occurring and the pipeline that is possibly causing it. filebeat-7.13.4-kibana-log-pipeline.

![image](https://user-images.githubusercontent.com/89588946/130971651-a3733856-9d1e-4910-bc53-02d3bbd46c71.png)
![image](https://user-images.githubusercontent.com/89588946/130971850-97011116-16c0-4ac7-95cf-0de67f7ec0cb.png)

We have tried to reindex with the script multiple times and tried to fix this pipeline to convert to type IP for source.address and source.ip. Nothing has fixed it yet.

**Comment by weslambert** [comment, marked as answer]
Yes, we have seen this issue with the Kibana index because of the way that templates were applied at one point, until they were switch to use the `so-*` pattern.  Are you using custom templates now? If you are, you'll need to update them to apply to `so-*`, as opposed to the individually defined index patterns (or add `so-kibana-*`, etc).

**Reply by Susfubar** [reply]
We have customized a template to add some fields but not any new templates. We dont even see a so-kibana-* template when viewing index management in kibana. 

We are running distributed setup. How would we update it so it will use the correct template?
